Client/Frontend/Browser/Toolbars/BottomToolbar/Menu/HistoryViewController.swift

```diff
@@ -48,12 +48,20 @@ class HistoryViewController: SiteTableViewController, ToolbarUrlActionsProtocol
         tableView.accessibilityIdentifier = "History List"
         title = Strings.historyScreenTitle
                 
-        navigationItem.rightBarButtonItem = UIBarButtonItem(image: #imageLiteral(resourceName: "playlist_delete_item").template, style: .done, target: self, action: #selector(performDeleteAll))
+        if !Preferences.Privacy.privateBrowsingOnly.value {
+            navigationItem.rightBarButtonItem =
+                UIBarButtonItem(image: #imageLiteral(resourceName: "playlist_delete_item").template, style: .done, target: self, action: #selector(performDeleteAll))
+        }
     }
     
     override func viewWillAppear(_ animated: Bool) {
         super.viewWillAppear(animated)
         
+        guard !Preferences.Privacy.privateBrowsingOnly.value else {
```

Comment:
convert to `if` this is too hard to read imo, two negations

Reply:
Actually I always try to `guard` the initial conditions instead of showing them inside If clause but this particular conditions, you are right it is very hard to read.

Change at https://github.com/brave/brave-ios/pull/4013/commits/1125af9d61d254a04478cfdd92ceaa3c0d8a8079